rhpam-kogito-operator.yaml

```diff
@@ -49,81 +49,54 @@ spec:
     name: v1
     schema:
       openAPIV3Schema:
-        description: KogitoBuild handles how to build a custom Kogito service in a
-          Kubernetes/OpenShift cluster.
+        description: KogitoBuild handles how to build a custom Kogito service in a Kubernetes/OpenShift cluster.
```

Comment:
I think we might consider deleting this file. Or would users install it manually from this git project?

Reply:
not sure, probably better keep it around for a while.